app.css

```diff
@@ -0,0 +1,56 @@
+* {
+  direction: rtl;
+  padding:0px;
+  margin:0px;
+}
+
+/*navbar*/
+
+#headWrapper{
```

Comment:
Why do you need an ID here?
Only use css ID when you must as it is globally exposed.